### PR TITLE
Add ten FOCI'24 issue 2 papers, two PETS'24 papers, and support accent characters in cite names. 

### PR DESCRIPTION
**Pull request by gfw-report**
This pull request contains two major components:

1. [Add ten FOCI'24 issue 2 papers.](https://github.com/NullHypothesis/censorbib/commit/0e2fa455c209d7dafe9e5f4af00b0c37e6c1d244)

* PETS publication page: https://www.petsymposium.org/foci/2024/
* FOCI program: https://foci.community/foci24.html#2024-summer-program
* Net4People post: https://github.com/net4people/bbs/issues/376

I do not include the three papers in Session III: Privacy Enhancing Technologies, as they are out of scope of Internet censorship.

---

2. [Support accent characters in cite names.](https://github.com/NullHypothesis/censorbib/commit/beb8b8aacad906b46aefeffa4a28a0797cbc5ed3)

A new commit introduces a cite name that contains an accent character:

> @inproceedings{Müller2024a,

It will cause the bibtex parser to fail with the following two errors:

> failed to extract cite name of: @inproceedings{Müller2024a,

and

> parse failed at 55:17: syntax error: unexpected $end, expecting tCOMM

We have both errors fixed in this commit. The second error is an upstream limitation, which I will try to get it merged to the upstream: https://github.com/nickng/bibtex.

### CODE REVIEW

**Comment by gfw-report** [comment]
> The second error is an upstream limitation, which I will try to get it merged to the upstream: https://github.com/nickng/bibtex.

I opened the pull request: https://github.com/nickng/bibtex/pull/25

File: references.bib [hunk 1]
```diff
@@ -1,3 +1,93 @@
+@inproceedings{Ahmed2024a,
+	author = {Mohamed Ahmed and Jeffrey Knockel},
+	title = {{Extended Abstract}: The Impact of Online Censorship on {LLMs}},
```
Comment (NullHypothesis) [inline references.bib:3]: I don't think we need to wrap these in brackets.
```suggestion
	title = {Extended Abstract: The Impact of Online Censorship on {LLMs}},
```

File: references.bib [hunk 2]
```diff
@@ -1,3 +1,93 @@
...
+	booktitle = {Free and Open Communications on the Internet},
+	publisher = {},
+	year = {2024},
+	url = {https://www.petsymposium.org/foci/2024/foci-2024-0006.pdf},
+}
+
+@inproceedings{Gao2024a,
+	author = {Tianyu Gao and Ping Ji},
+	title = {{Extended Abstract}: Leveraging Large Language Models to Identify {Internet} Censorship through Network Data},
```
Comment (NullHypothesis) [inline references.bib:12]:
```suggestion
	title = {Extended Abstract: Leveraging Large Language Models to Identify {Internet} Censorship through Network Data},
```

File: src/main.go [hunk 3]
```diff
@@ -13,8 +13,9 @@ import (
 	"github.com/nickng/bibtex"
 )
 
-// Matches e.g.: @inproceedings{Doe2024a,
-var re = regexp.MustCompile(`@[a-z]*\{([A-Za-z\-]*[0-9]{4}[a-z]),`)
+// Matches e.g.: @inproceedings{Müller2024a,
+// \p{L}\p{M} matches any letter, including accented characters.
+var re = regexp.MustCompile(`@[a-z]*\{([\"\p{L}\p{M}\-]*[0-9]{4}[a-z]),`)
```
Comment (NullHypothesis) [inline src/main.go:18]: Nice fix! :+1:

**Review by NullHypothesis — CHANGES_REQUESTED** [review]
One minor comment but looks good! Thanks for your work!

**Comment by gfw-report** [comment]
I have now removed `{}` from Extended Abstract. 

Thank you so much for your review, @NullHypothesis !

**Comment by gfw-report** [comment]
I added two PETS'24 papers in this PR.

**Review by NullHypothesis — APPROVED** [review]
Noice noice, looks great!

**Pull request merged**